charms/istio-pilot/actions.yaml

```diff
@@ -0,0 +1,21 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+save-tls-secret:
```

Comment:
I also thought of that, but I did not want this to be confused with configuring TLS certs via a TLS provider, that's why I wanted `save-tls-secret`, but you are right that this is internal to the charm and from the user perspective this is a configuration of TLS, rather.
I'm not sure if `configure-tls-certs` is appropriate though, it could be `configure-tls-manually`(?)

Reply:
`configure-tls-manually` works for me to differentiate between the relations way of configuring